jsonnet/kube-prometheus/rules/general.libsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'kube-prometheus-general.rules',
+        rules: [
+          {
+            expr: 'count (up == 1) without (instance, pod, node)',
+            record: 'sum:up1',
```

Comment:
sorry, this was my mistake when I suggested this, this should be `count:up1`, as we're aggregating by counting instead of summing